Content.Client/GameObjects/Components/PlaceableSurfaceComponent.cs

```diff
@@ -1,31 +1,64 @@
 #nullable enable
 using Content.Shared.GameObjects.Components;
 using Robust.Shared.GameObjects;
+using Robust.Shared.Maths;
 
 namespace Content.Client.GameObjects.Components
 {
     [RegisterComponent]
     [ComponentReference(typeof(SharedPlaceableSurfaceComponent))]
     public class PlaceableSurfaceComponent : SharedPlaceableSurfaceComponent
     {
-        private bool _isPlaceable;
+        private bool isPlaceable;
+        private bool placeCentered;
+        private Vector2 positionOffset;
 
         public override bool IsPlaceable
         {
-            get => _isPlaceable;
+            get => isPlaceable;
             set
             {
-                if (_isPlaceable == value)
+                if (isPlaceable == value)
                 {
                     return;
                 }
 
-                _isPlaceable = value;
+                isPlaceable = value;
 
                 Dirty();
```

Comment:
```suggestion
```

No point calling Dirty on the client

Reply:
Oh crap, I committed the suggestion from github, then overwrote with my local copy I think, sorry about that

Reply:
It should be fixed now.